server/src/entity/Task.ts

```diff
@@ -12,6 +12,9 @@ export class Task {
     @Column({ default: false })
     completed: boolean;
 
+    @CreateDateColumn()
```

Comment:
```
@CreateDateColumn is a special column that is automatically set to the entity's insertion date. You don't need to set this column - it will be automatically set.
```
這個會在寫入資料的時候自動寫入時間喔，但剛寫入應該不能有完成時間

Reply:
了解，已修改為 @Column

Reply:
了解，已修改為 column 設定為 default